src/helpers.js

```diff
@@ -11,19 +11,26 @@ function transposeDistance(transposeKey, songKey) {
 }
 
 /* eslint import/prefer-default-export: 0 */
-export function renderChord(chord, lineKey, songKey, transposeKey) {
-  const distance = presence(transposeKey) ? transposeDistance(transposeKey, songKey) : 0;
+export function renderChord(chord, lineKey, songKey, transposeKey, capo) {
   let chordObj = Chord.parse(chord);
 
   if (!chordObj) {
     return chord;
   }
 
-  if (presence(transposeKey) && presence(songKey)) {
-    chordObj = chordObj.transpose(distance).useModifier(transposeKey.modifier);
+  let transpose = -1 * (capo || 0);
```

Comment:
Can you explain what this does? You pass the capo value into the renderChord function and then it will multiply -1 by the capo value?